spec/features/user_adds_a_review_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+feature "user sees a specific bookstore page" do
+  zip = "18900"
```

Comment:
Hmm, why are you doing this? Also, why not use a factory? I presume you'll have to make lots of these going forward!